server/src/main/java/io/spine/server/integration/IntegrationBroker.java

```diff
@@ -213,72 +187,24 @@ public void register(EventDispatcher dispatcher) {
      *         the dispatcher to unregister
      */
     public void unregister(EventDispatcher dispatcher) {
-        Iterable<EventClass> transformed = dispatcher.externalEventClasses();
-        for (EventClass cls : transformed) {
-            ChannelId channelId = toChannelId(cls);
-            Subscriber subscriber = subscriberHub.get(channelId);
-            ExternalMessageObserver observer = observerFor(cls);
-            subscriber.removeObserver(observer);
-        }
+        events.unregister(dispatcher);
         subscriberHub.closeStaleChannels();
     }
 
-    private static ChannelId toChannelId(EventClass cls) {
-        TypeUrl targetType = cls.typeUrl();
-        return channelIdFor(targetType);
-    }
-
-    private ExternalMessageObserver observerFor(EventClass externalClass) {
-        ExternalMessageObserver observer =
-                new ExternalMessageObserver(context, externalClass.value(), this);
-        return observer;
-    }
-
-    /**
-     * Notifies other Bounded Contexts that this integration bus instance now requests a different
-     * set of message types.
-     *
-     * <p>Sends out an instance of {@linkplain RequestForExternalMessages
-     * request for external messages} for that purpose.
-     */
-    private void notifyTypesChanged() {
-        ImmutableSet<ExternalMessageType> needs = subscriberHub
-                .ids()
-                .stream()
-                .map(channelId -> ExternalMessageType
-                        .newBuilder()
-                        .setMessageTypeUrl(channelId.getTargetType())
-                        .setWrapperTypeUrl(EVENT.value())
-                        .buildPartial())
-                .collect(toImmutableSet());
-        broadcast.onTypesChanged(needs);
-    }
-
-    /**
-     * Notifies other Bounded Contexts of the application about the types requested by this Context.
-     *
-     * <p>The {@code IntegrationBroker} sends a {@link RequestForExternalMessages}. The request
-     * triggers other Contexts to send their requests. As the result, all the Contexts know about
-     * the needs of all the Contexts.
-     */
-    void notifyOthers() {
-        broadcast.send();
-    }
-
     /**
      * Removes all subscriptions and closes all the underlying transport channels.
      */
     @Override
     public void close() throws Exception {
-        configurationObserver.close();
-        notifyTypesChanged();
-
+        if (config != null) {
+            config.close();
+        }
         subscriberHub.close();
         publisherHub.close();
```

Comment:
`IntegrationBroker` seems not to be a main holder and user of these resources. Could we drop these fields from this class and store `TransportLink : Closeable` instead?